nutils/evaluable.py

```diff
@@ -3668,6 +3668,11 @@ def exp(x):
 def ln(x):
   return Log(x)
 
+def divmod(x, y):
+  div = FloorDivide(*_numpy_align(x, y))
+  mod = x - div * y
```

Comment:
If the reason of not using function `mod` or class `Mod` defined below is to have some sort of canonical representation, shall we then also reimplement `def mod(arg1, arg2):` using `divmod(arg1, arg2)[1]`?

Reply:
I'm not sure.. removing `evaluable.Mod` means repeating this logic in function. Maybe it's better to use `Mod` after all, or keep things local to `divmod` as proposed, from the reasoning that divmod likely flags a situation in which div and mod are going to be evaluated together.